src/Migrations/Version20200312181539.php

```diff
@@ -37,16 +37,16 @@ public function up(Schema $schema) : void
+        $this->addSql('CREATE TABLE navigation_timings (page_view_id INT UNSIGNED AUTO_INCREMENT NOT NULL, dns_duration SMALLINT UNSIGNED NOT NULL, connect_duration SMALLINT UNSIGNED NOT NULL, ttfb SMALLINT NOT NULL, download_time SMALLINT NOT NULL, first_byte SMALLINT UNSIGNED NOT NULL, redirect_duration SMALLINT UNSIGNED NOT NULL, last_byte_duration SMALLINT UNSIGNED NOT NULL, first_paint SMALLINT UNSIGNED NOT NULL, first_contentful_paint SMALLINT UNSIGNED NOT NULL, redirects_count SMALLINT NOT NULL, total_img_size INT NOT NULL, total_js_compressed_size INT NOT NULL, total_js_uncomressed_size INT NOT NULL, total_css_compressed_size INT NOT NULL, total_css_uncomressed_size INT NOT NULL, number_js_files SMALLINT NOT NULL, number_css_files SMALLINT NOT NULL, number_img_files SMALLINT NOT NULL, url_id INT UNSIGNED NOT NULL, user_agent_id INT UNSIGNED NOT NULL, device_type_id INT NOT NULL, os_id INT NOT NULL, process_id CHAR(8) NOT NULL COLLATE utf8mb4_unicode_ci, guid CHAR(128) NOT NULL COLLATE utf8mb4_unicode_ci, stay_on_page_time SMALLINT UNSIGNED NOT NULL, created_at DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP, load_event_end SMALLINT UNSIGNED NOT NULL, INDEX created_at (created_at), INDEX url_id_2 (url_id, created_at), INDEX device_type_id (device_type_id), INDEX url_id (url_id), INDEX page_view_id (page_view_id, user_agent_id), INDEX user_agent_id (user_agent_id), INDEX os_id (os_id), INDEX guid (guid), PRIMARY KEY(page_view_id)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB COMMENT = \'\' ');
```

Comment:
I think that we should not have a migration in this MR as it is only about php cs fixer.